File: program/program/convert-temperature-from-celsius-to-kelvin/convert_temperature_from_celsius_to_kelvin.go
package main

import "fmt"

func celsiusToKelvin(celsius float64) float64 {
    return celsius + 273.15
}

func main() {
    celsius := -40
    kelvin := celsiusToKelvin(celsius)

    fmt.Println("Input (C):", celsius)
    fmt.Println("Output (K):", kelvin)

}

File: program/program/convert-temperature-from-celsius-to-kelvin/convert_temperature_from_celsius_to_kelvin.rs
// Convert celsius to kelvin
use std::env;
fn main() {
    let args: Vec<String> = env::args().collect();
    let celsius = &args[1];
    let celsius: f32 = celsius.parse().expect("Invalid value for celcius");
    let kelvin = celsius + 273.15;
    println!("Input ( C ) : {}", celsius);
    println!("Output ( K ) : {}", kelvin);
}
